doc/nrf/protocols/wifi/sap_mode/sap.rst

```diff
@@ -113,8 +115,3 @@ For a typical SAP mode operation, the nRF70 Series device acts as a virtual rout
 The SAP interface needs to be configured with a valid IP address and a subnet mask.
 Without this configuration, the Wi-Fi station device might disconnect from the SAP interface.
 But the station devices use :abbr:`DHCP (Dynamic Host Configuration Protocol)` to get an IP address from the virtual router.
-Currently, the Zephyr networking stack does not support DHCP server functionality.
```

Comment:
we can link to https://docs.zephyrproject.org/apidoc/latest/group__dhcpv4__server.html